README.md

```diff
@@ -1,2 +1,114 @@
-# doppler-html-editor-api
-API Rest from Doppler HTML Editor
+# Doppler HTML Editor API
+
+## Campaigns
+
+```http
+GET /accounts/{accountName}/campaigns/{campaignId}/content/design
+```
+
+Response:
+```json
+{
+	"counters": {},
+	"body": {},
+	"schemaVersion": 6
+}
+```
+
+## Thumbnails
+
+```http
```

Comment:
does `http` work?

Reply:
Yes, it is!

https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L2373